WordPress/Classes/ViewRelated/Post/FeaturedImageViewController.m

```diff
@@ -22,13 +22,12 @@ - (void)dealloc
 
 - (id)initWithPost:(AbstractPost *)post
 {
-    self = [super init];
+    self = [super initWithImage:nil andURL:[NSURL URLWithString:post.featuredImage.remoteURL]];
     if (self) {
         self.title = NSLocalizedString(@"Featured Image", @"");
```

Comment:
Very nitpicky comment, but if this is a clean up PR it might be nice to add any missing comments where we localize.